### PR TITLE
Multiassay

### PR DESCRIPTION
**Pull request by antagomir**
I started to list the available options for representing multiassay/multitable data. It might be good work out a set of recommendations on how to represent differen scenarios. We have at least 

1. transformations (counts, relabundance, clr etc) -> assays()
2. alternative experiments (e.g 16S vs metagenomics for taxonomic abundances) -> assays() or altExp()
3. multiple experiments (e.g. taxonomic abundances vs. metaT or metabolomics) -> assays() or altExp() or MAE.

### CODE REVIEW

**Comment by FelixErnst** [comment]
I made some addition. Let me know what you think.

I am happy to merge it if you give the go.

**Review by FelixErnst — APPROVED** [review]
Its a good little chapter 😄 

I made some changes. Let me know what you changed.

I am happy to merge if you give the go.

**Comment by antagomir** [comment]
Ok I made some improvements + additions, it should help to have some basis for how to approach the various multi-table options, now we have.

File: 18-multitable.Rmd [hunk 1]
```diff
@@ -1,11 +1,126 @@
...
+
+
+For information have a look at the [intro vignette](https://bioconductor.org/packages/release/bioc/vignettes/MultiAssayExperiment/inst/doc/MultiAssayExperiment.html) of the `MultiAssayExperiment` package.  
+
+ 
+   Option   Rows (samples)    Cols (features)               Recommended  
+---------   --------------    ---------------  ------------------------
+   assays  	     match              match      Data transformations             
+   altExp            match              free    Alternative experiments   
+MultiAssay         mapping              free     Multi-omic experiments
```
Comment (FelixErnst) [inline 18-multitable.Rmd:121]: This is the other way around. columns are samples and rows are features. SummarizedExperiment doesn't actually care, but SingleCellExperiment and TreeSummarizedExperiment do

Comment (antagomir) [reply 18-multitable.Rmd]: Oh yes, this was just a bug, fixed now.

**Pull request merged**